lib/manager/actions/feeds.js

```diff
@@ -81,6 +82,10 @@ export function createFeedSource (newFeed: NewFeed) {
 export function updateFeedSource (feedSource: Feed, properties: {[string]: any}) {
   return function (dispatch: dispatchFn, getState: getStateFn) {
     dispatch(savingFeedSource())
+    // Convert all labels to be only strings, which the server requires
+    feedSource.labels = stringifyLabels(feedSource.labels)
+    properties.labels = stringifyLabels(properties.labels)
```

Comment:
Please remove if refactoring data model as recommended in https://github.com/ibi-group/datatools-server/pull/403#pullrequestreview-714190079.